### PR TITLE
Style detail page

### PR DESCRIPTION
**Pull request by veredrec**
Relates #94 

- Fix bug with detail page (with `selected`)
- Add style to detail page

### CODE REVIEW

File: public/scss/_detail.scss [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+.grade, .salary {
+  margin-top: -0.5em;
+}
+.last {
+  padding: 0.2em;
+}
+.line {
+  border: 0;
+  height: 1px;
```
Comment (Karyum) [inline public/scss/_detail.scss]: pixel?

http://gph.is/1jE2Wzg

Comment (veredrec) [reply public/scss/_detail.scss]: It's totally a mistake :flushed: I'm changing it!

Comment (veredrec) [reply public/scss/_detail.scss]: And the gif is great :laughing:

Comment (veredrec) [reply public/scss/_detail.scss]: @Karyum changed it :)

**Pull request merged**